sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobAsyncClientBase.java

```diff
@@ -545,27 +546,43 @@ public boolean isSnapshot() {
      * @throws IllegalArgumentException If {@code blockSize} is less than 0 or greater than 100MB.
      * @throws UncheckedIOException If an I/O error occurs.
      */
-    public Mono<Void> downloadToFile(String filePath, BlobRange range, Integer blockSize,
-        ReliableDownloadOptions options, BlobAccessConditions accessConditions, boolean rangeGetContentMD5) {
-        return withContext(context -> downloadToFile(filePath, range, blockSize, options, accessConditions,
+    public Mono<Response<BlobProperties>> downloadToFileWithResponse(String filePath, BlobRange range,
+                                                                     Integer blockSize, ReliableDownloadOptions options,
```

Comment:
This comment applies. This is mushed against the right margins.
https://github.com/Azure/azure-sdk-for-java/pull/5547#discussion_r329232537

Reply:
I think a lot of these have really long names and the indentation should be fixed in this class. These params are hard to read because they're all on new lines.

Reply:
Thanks for the link. Changed the indentation.